docs/esa/iso/iso.rst

```diff
@@ -202,55 +209,54 @@ This will download the ISO postcard for the observation '80001538' and it will s
 3. Getting ISO Tables Details through TAP
 ------------------------------------------
 
-.. code-block:: python
+.. doctest-remote-data::
 
   >>> from astroquery.esa.iso import ISO
-  >>> ISO.get_tables()
-  
+  >>> ISO.get_tables()    # doctest: +IGNORE_OUTPUT
   INFO: Retrieving tables... [astroquery.utils.tap.core]
   INFO: Parsing tables... [astroquery.utils.tap.core]
   INFO: Done. [astroquery.utils.tap.core]
-  ['hpdp.cam_sato', 'hpdp.cambendo', 'hpdp.chopc2i', 'hpdp.chopc2ii', 'hpdp.compkon', 
-  'hpdp.evolkon', 'hpdp.extrakon', 'hpdp.kon3p6i', 'hpdp.kon3p6ii', 'hpdp.konkoly', 
+  ['hpdp.cam_sato', 'hpdp.cambendo', 'hpdp.chopc2i', 'hpdp.chopc2ii', 'hpdp.compkon',
+  'hpdp.evolkon','hpdp.extrakon', 'hpdp.kon3p6i', 'hpdp.kon3p6ii', 'hpdp.konkoly',
   'hpdp.lwsasti', 'hpdp.lwsastii', 'hpdp.misckon', 'hpdp.p32_c200', 'hpdp.p32virgo',
-  'hpdp.scanskon', 'hpdp.sloansws', 'hpdp.solarkon', 'hpdp.sws01hrd', 'hpdp.ysokon',
-  'tap_config.coord_sys', 'ida.dqr', 'ida.dqrflags', 'ida.footprints', 'ida.fov_info', 
-  'ida.iasd_line', 'ida.iso_wcs_info', 'ida.observers', 'ida.postcards', 'ida.proposals',
-  'ida.usredata', 'ida.usredata_files', 'ida.object_type', 'ida.observations', 
-  'tap_schema.columns', 'tap_schema.key_columns', 'tap_schema.keys', 'tap_schema.schemas', 
-  'tap_schema.tables', 'ivoa.obscore',
-  'ivoa.siap', 'ivoa.ssap']
+  'hpdp.scanskon', 'hpdp.sloansws', 'hpdp.solarkon','hpdp.sws01hrd', 'hpdp.ysokon',
+  'tap_config.coord_sys', 'tap_schema.columns', 'tap_schema.key_columns','tap_schema.keys',
+  'tap_schema.schemas', 'tap_schema.tables', 'ida.aph', 'ida.cam_expert', 'ida.cam_measurement',
+  'ida.dqr', 'ida.dqrflags', 'ida.iasd_line', 'ida.lws_expert', 'ida.lws_extras', 'ida.lws_lsan',
+  'ida.lws_measurement','ida.obs_pointing', 'ida.observers', 'ida.pht_expert', 'ida.postcards',
+  'ida.proposals', 'ida.publication_authors', 'ida.sws_common', 'ida.sws_expert', 'ida.usredata',
+  'ida.usredata_files', 'ida.object_type', 'ida.observations', 'ida.publications', 'ida.publications_observations',
+  'ivoa.obscore', 'ivoa.siap', 'ivoa.ssap']
 
 This will show the available tables in ISO TAP service in the ISO Data Archive.
 
 -------------------------------------
 4. Getting columns details of ISO TAP
 -------------------------------------
 
-.. code-block:: python
-
-  >>> from astroquery.esa.iso import ISO
-  >>> ISO.get_columns('ida.observations')
+.. doctest-remote-data::
 
+  >>> from astroquery.esa.iso import ISO 
+  >>> ISO.get_columns('ida.observations')     # doctest: +IGNORE_OUTPUT
```

Comment:
I wonder how stable this output is, maybe we can try and not ignore it right now, but only put the ignore on it if it's proven to be changing often

Reply:
I also did the `get_tables` one.